uw-frame-components/portal/widgets/partials/widget-card.html

```diff
@@ -0,0 +1,87 @@
+<md-card class="widget-frame" id="widget-id-{{ widget.nodeId }}" aria-label="{{ widget.title }} widget">
+
+  <!-- MAINTENANCE MODE OVERLAY -->
+  <div class="overlay__maintenance-mode" ng-if="widget.lifecycleState === 'MAINTENANCE'">
+    <div class="maintenance-content">
+      <p>
+        <md-icon class="md-warn">warning</md-icon>
+      </p>
+      <p>This app is undergoing maintenance and is temporarily unavailable.</p>
+    </div>
+  </div>
+
+  <!-- HEADER -->
+  <md-card-header class="widget-header">
+    <!-- Widget Chrome -->
+    <md-button class="widget-action widget-info md-icon-button" aria-label="Description: {{ widget.description }}" role="tooltip" ng-hide="widget.lifecycleState === 'MAINTENANCE'">
+      <md-tooltip md-direction="top" class="widget-action-tooltip">
+        {{ widget.description }}
+      </md-tooltip>
+      <md-icon>info</md-icon>
+    </md-button>
+    <md-button class="widget-action widget-remove md-icon-button"
+               aria-label="remove {{ widget.title }} widget from your home screen"
+               ng-click="widgetCtrl.removewidget(widget.nodeId, widget.title)"
+               ng-hide="GuestMode || cantRemove || widget.lifecycleState === 'MAINTENANCE'">
+      <md-icon>close</md-icon>
+    </md-button>
+
+    <md-card-header-text>
+      <span class="md-title" style="text-align: center;" aria-label="{{ widget.title }}" tabindex="0">
+	      {{ widget.title }}
+      </span>
+    </md-card-header-text>
+  </md-card-header>
+
+  <!-- BODY -->
+  <md-card-content class="widget-content">
+
+    <div ng-switch="widgetType" class="widget-type-container">
+
+      <div ng-switch-when="option-link">
+        <option-link app="widget" config="widget.widgetConfig"></option-link>
+      </div>
+
+      <div ng-switch-when="weather">
+        <weather-widget app="widget" config="widget.widgetConfig"></weather-widget>
+      </div>
+
+      <div ng-switch-when="rss">
+        <rss-widget app="widget" config="widget.widgetConfig"></rss-widget>
+      </div>
+
+      <div ng-switch-when="list-of-links">
+        <list-of-links app="widget" config="widget.widgetConfig"></list-of-links>
+      </div>
+
+      <div ng-switch-when="search-with-links">
+        <search-with-links app="widget" config="widget.widgetConfig"></search-with-links>
+      </div>
+
+      <div ng-switch-when="custom" class="custom-widget">
+        <div ng-controller="CustomWidgetController as customWidgetCtrl">
+          <div ng-if="loading" layout="row" layout-align="center center">
+            <loading-gif data-object='content' data-empty='isEmpty'></loading-gif>
+          </div>
+          <content-item ng-if="!loading"></content-item>
+        </div>
+      </div>
+
+      <div ng-switch-when="BASIC">
+        <a tabindex="-1" ng-href="{{ renderUrl() }}" target="{{ widget.target }}" class="basic-widget">
```

Comment:
Could this be more secure if it included [`rel="noopener"`](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)?